be/src/runtime/tablets_channel.cpp

```diff
@@ -476,10 +476,9 @@ Status TabletsChannel::add_batch(const TabletWriterAddRequest& request,
         if (!st.ok()) {
             auto err_msg = strings::Substitute(
                     "tablet writer write failed, tablet_id=$0, txn_id=$1, err=$2"
-                    ", errcode=$3, msg:$4",
-                    tablet_to_rowidxs_it.first, _txn_id, st.code(), st.precise_code(),
-                    st.get_error_msg());
-            LOG(WARNING) << err_msg;
+                    ", precise_code=$3",
+                    tablet_to_rowidxs_it.first, _txn_id, st.code(), st.precise_code());
+            LOG(WARNING) << err_msg << " bt:" << st.get_error_msg();
```

Comment:
Why would `get_error_msg()` print stack trace?

Reply:
OLAP_INTERNAL_ERROR uses stacktrace to describe error details.